config/confd/templates/server.conf.tmpl

```diff
@@ -11,7 +11,12 @@ persist-tun
 
 proto tcp4-server
 compress lzo
-keepalive {{ getenv "VPN_KEEPALIVE_FREQUENCY" "10" }} {{ getenv "VPN_KEEPALIVE_TIMEOUT" "60" }}
+
+# Tell clients to exit on keepalive timeouts
+ping {{ getenv "VPN_KEEPALIVE_FREQUENCY" "10" }}                    # interval
+ping-exit {{ (atoi (getenv "VPN_KEEPALIVE_TIMEOUT" "60")) mul 2 }}  # timeout*2
```

Comment:
Do we actually want/need `ping-exit` on the server side? I know that `keepalive` was setting `ping-restart` but I couldn't tell from the docs if we actually want the server to use `ping-exit` or stick with `ping-restart` or neither?

Reply:
I think both should match

Reply:
The docs mention how `ping-restart` is handled on the server side:

"In server mode, --ping-restart, --inactive or any other type of internally generated signal will always be applied to individual client instance objects, never to whole server itself. Note also in server mode that any internally generated signal which would normally cause a restart, will cause the deletion of the client instance object instead." 

However nothing is being mentioned for `ping-exit` for the server mode. My gut feeling is that we need to keep `ping-restart` on the server side.

Reply:
`ping-exit` on the server may indeed have undesired behavior on the server side: https://docs.netgate.com/pfsense/en/latest/vpn/openvpn/configure-server-ping.html#ping-method